app/controllers/v0/sessions_controller.rb

```diff
@@ -84,7 +84,7 @@ def saml_user
 
     def create_saml_user
       user = User.new(user_attributes)
-      user = Decorators::MviUserDecorator.new(user).create unless user.loa1?
+      user = Decorators::MviUserDecorator.new(user).create unless user.loa1? # || user.gender.nil?
```

Comment:
i added this as a comment just to bring attention to it. 

@kreek does it make sense to try and decorate the user if gender is known to be `nil`?